tardis/spectrum/base.py

```diff
@@ -178,6 +182,27 @@ def calculate_reabsorbed_luminosity(
             luminosity_wavelength_filter
         ].sum()
 
+    def solve(self, transport_state):
```

Comment:
What is this function supposed to be doing?

Reply:
Is the docstring not clear enough? It computes the spectra and returns them as a tuple of TARDISSpectrum objects.